duckduckgo/Android
# Added pixels to obtain usage data about overflow menu

Task/Issue URL: https://app.asana.com/0/715106103902962/1199553549676118/f
Tech Design URL: 
CC: 

**Description**:
Added pixels to the overflow menu items in the browser page to understand how users interact with them and take informed decisions in the future

**Steps to test this PR**:
1. Open app
2. Interact with menu items
3. See corresponding added pixels firing


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
